### ISSUE TITLE
[BUG]: Graphical bugs in Bleach blade battlers 2nd

### ISSUE THREAD

**Issue by LuismaSP89** [PCSX2/pcsx2#5299]
### Describe the Bug

Graphical bugs when the characters do the "Bankai" in the bleach blade battlers 2nd. Tested with Opengl, Directx 11, & Vulkan backends, all with the same problem. It happens most of the time, but sometimes don't.

### Reproduction Steps

On bleach game: Start any fight, for example 1 vs 1 free battle. Then, during the fight, when you have enough power, press L1 + R1 to do the bankai, you'll see the affected scene.

### PCSX2 Revision

1.7.2212

### Operating System

Windows 11

### CPU

i74790k

### GPU

GTX 970

### GS Window Screenshots

![gs_20220109164718](https://user-images.githubusercontent.com/12205048/148690547-684b64f0-838e-4ca7-a748-653ff06221da.jpg)
![gs_20220109163805](https://user-images.githubusercontent.com/12205048/148690548-62cbf7a8-d097-4c8c-aacf-6d1fa99721ff.jpg)
![gs_20220109164329](https://user-images.githubusercontent.com/12205048/148690551-8ae9a2f8-9d5a-4b5f-a16c-0d6f12917ec6.jpg)

**Comment by lightningterror** [comment]
A gsdump and a blockdump will be nice.

**Comment by LuismaSP89** [comment]
It's very strange, because I tried like 20 times more, and I'm no longer able to reproduce the duplicated image issue. The only thing I changed is the blending accurate from full to normal. I don't know if this makes sense.

In any case, the other graphical bugs (Like the 2nd image of the first post with all the garbage in the screen after the bankai scene) still happening, also a bug with the Rukia bankai that makes the screen white during the bankai itself.

Attached 2 GSdump for this 2 problems, with the blockdump option activated.

Thanks a lot for your help!

[snaps.zip](https://github.com/PCSX2/pcsx2/files/7835686/snaps.zip)
.
